.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+jobs:
+  build:
+    docker:
+      - image: node:18.14.2
+    environment: 
+      repo_token: qyL7VHpxdSYwaWsAhyNjePEGUWcmSegos
```

Comment:
Do not provide the secrets to the public. 
Create Environment variables under github Secrets, Add secrete name and it's value